src/assets/i18n/messages.de.json

```diff
@@ -0,0 +1,198 @@
+  "6a3777f913cf3f288664f0632b9f24794fdcc24e": "Abbrechen",
+  "322ed150e02666fe2259c5b4614eac7066f4ffa0": "Erweitert",
+  "b7ffe7c6586d6f3f18a9246806a7c7d5538ab43e": "Simulierter Befehl:",
+  "4e4c721129466be9c3862294dc40241b64045998": "Benutzerdefinierte Argumente verwenden",
+  "ad2f8ac8b7de7945b80c8e424484da94e597125f": "Benutzerdefinierte Argumente",
+  "a6911c2157f1b775284bbe9654ce5eb30cf45d7f": "Der FQDN muss nicht angegeben werden, sondern nur der Teil danach. Argumente werden wie folgt getrennt: ,,",
+  "3a92a3443c65a52f37ca7efb8f453b35dbefbf29": "Benutzerdefinierte Ausgabe verwenden",
+  "d9c02face477f2f9cdaae318ccee5f89856851fb": "Benutzerdefinierte Ausgabe",
+  "fcfd4675b4c90f08d18d3abede9a9a4dff4cfdc7": "Dokumentation",
+  "19d1ae64d94d28a29b2c57ae8671aace906b5401": "Der Pfad ist relativ zum Konfigurations-Download-Pfad. Dateiendung nicht angeben.",
+  "8fad10737d3e3735a6699a4d89cbf6c20f6bb55f": "Authentifizierung verwenden",
+  "08c74dc9762957593b91f6eb5d65efdfc975bf48": "Benutzername",
+  "c32ef07f8803a223a83ed17024b38e8d82292407": "Passwort",
+  "4a0dada6e841a425de3e5006e6a04df26c644fa5": "Audio",
+  "9779715ac05308973d8f1c8658b29b986e92450f": "Ihre Audiodateien befinden sich hier",
+  "47546e45bbb476baaaad38244db444c427ddc502": "Playlisten",
+  "78bd81adb4609b68cfa4c589222bdc233ba1faaa": "Keine Wiedergabelisten verfügbar. Erstellen Sie eine aus Ihren heruntergeladenen Audiodateien, indem Sie auf das blaue Pluszeichen klicken.",
+  "9d2b62bb0b91e2e17fb4177a7e3d6756a2e6ee33": "Video",
+  "960582a8b9d7942716866ecfb7718309728f2916": "Ihre Videodateien befinden sich hier",
+  "0f59c46ca29e9725898093c9ea6b586730d0624e": "Keine Playlisten verfügbar. Erstellen Sie eine aus heruntergeladenen Audiodateien, indem Sie auf das blaue Pluszeichen klicken.",
+  "616e206cb4f25bd5885fc35925365e43cf5fb929": "Name:",
+  "c52db455cca9109ee47e1a612c3f4117c09eb71b": "URL:",
+  "c6eb45d085384903e53ab001a3513d1de6a1dbac": "Kanal:",
+  "109c6f4a5e46efb933612ededfaf52a13178b7e0": "Dateigröße:",
+  "bd630d8669b16e5f264ec4649d9b469fe03e5ff4": "Pfad:",
+  "a67e7d843cef735c79d5ef1c8ba4af3e758912bb": "Hochgeladen am:",
+  "f4e529ae5ffd73001d1ff4bbdeeb0a72e342e5c8": "Schließen",
+  "ca3dbbc7f3e011bffe32a10a3ea45cc84f30ecf1": "ID:",
+  "e684046d73bcee88e82f7ff01e2852789a05fc32": "Anzahl:",
+  "321e4419a943044e674beb55b8039f42a9761ca5": "Info",
+  "826b25211922a1b46436589233cb6f1a163d89b7": "Löschen",
+  "34504b488c24c27e68089be549f0eeae6ebaf30b": "Löschen und zur Blacklist hinzufügen",
+  "121cc5391cd2a5115bc2b3160379ee5b36cd7716": "Einstellungen",
+  "801b98c6f02fe3b32f6afa3ee854c99ed83474e6": "URL",
+  "54c512cca1923ab72faf1a0bd98d3d172469629a": "URL, über die auf diese Applikation zugegriffen wird, ohne den Port.",
+  "cb2741a46e3560f6bc6dfd99d385e86b08b26d72": "Port",
+  "22e8f1d0423a3b784fe40fab187b92c06541b577": "Der gewünschte Port. Standard ist 17442.",
+  "d4477669a560750d2064051a510ef4d7679e2f3e": "Multi-user",
```

Comment:
```suggestion
  "d4477669a560750d2064051a510ef4d7679e2f3e": "Multi-User",
```

Reply:
Multi-Benutzer klingt ziemlich doof 😋

Reply:
Ich hab noch Modus angehängt. Den im original heißt es `Multi-user mode`

Reply:
Multiuser wäre glaube ich die "korrekte" deutsche Schreibweise, allerdings bin auch ich für die Trennung mit Bindestrich für bessere Lesbarkeit. So ist das Wort auch sonst in deutschen IT-Publikationen anzutreffen.

Multi-Benutzer klingt ein bisschen nach "Rechenmaschine", wenn ihr wisst was ich meine. :P

Reply:
Jo, seid ihr also mit `Multi-User Modus` einverstanden?